eo-runtime/src/test/eo/org/eolang/cti-tests.eo

```diff
@@ -8,6 +8,7 @@
 +unlint sparse-decoration
 +unlint cti
 +unlint incorrect-unlint
++unlint unlint-non-existing-defect
```

Comment:
@h1alexbel why add here?

Reply:
@maxonfjvipon since we unlinting `cti`, which is does not exists in `unlint-non-existing-defect` context:

```text
Unlinting rule 'cti' doesn't make sense, since there are no defects with it (unlint-non-existing-defect)
```

Reply:
@maxonfjvipon I don't think so, since there is no lint with name `cti`, thats why we get the warning